app/assets/javascripts/add-playlist-option.js.coffee

```diff
@@ -32,7 +32,7 @@ $ ->
       has_new_opt = true
   
   if !has_new_opt
-    select_element.prepend(new Option(addnew))
+    select_element.append(new Option(addnew))
```

Comment:
Does this change the location of the "Add new playlist" option to the bottom of the dropdown list?

Reply:
No, it doesn't